includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -1174,10 +1174,29 @@ public function add_last_activity_to_user_query( WP_User_Query $query ) {
 			WHERE {$wpdb->comments}.user_id = {$wpdb->users}.ID
 			AND {$wpdb->comments}.comment_approved IN ('complete', 'passed', 'graded')
 			AND {$wpdb->comments}.comment_type = 'sensei_lesson_status'
-			ORDER BY {$wpdb->comments}.comment_date_gmt DESC
 		) AS last_activity_date";
 	}
 
+	/**
+	 * Order query based on the custom field.
+	 *
+	 * @since  4.3.0
+	 * @access public
+	 *
+	 * @param WP_User_Query $query The user query.
+	 */
+	public function add_orderby_custom_field_to_query( WP_User_Query $query ) {
+		global $wpdb;
+		if ( ! isset( $query->query_vars['orderby'] ) || ! isset( $query->query_vars['order'] ) ) {
+			return '';
+		}
+		$query->query_orderby = sprintf(
+			'ORDER BY %s %s',
+			esc_sql( $query->query_vars['orderby'] ),
+			esc_sql( $query->query_vars['order'] )
+		);
```

Comment:
I think it's better to follow the standard approach and use `$wpdb->prepare` instead of `sprintf` + `esc_sql`.

Reply:
It actually can't work with $wpdb->prepare since it adds the quotes and then the query is not working. I can't find the post on this topic now.

Reply:
yeah, good idea I changed it: [here](https://github.com/Automattic/sensei/pull/4960/commits/d2545673ad2072a463f4ba1fd2f18a9fe63ecfe2)